server.js

```diff
@@ -52,92 +87,104 @@ app.get('/api/profile', function (req, res) {
-          name: "name4",
-          description: "description4"
+          name: "Outdoor Adventure",
+          description: "Not all the time its about going to specific places with goal in mind. There are times when my friends and I take on unknown hiking trail and explore random cities and suburban"
         }],
     }
   })
 });
 
 // Api template for images
+// sample_background are images I found online thats interesting
+// LA_trip has actual images I took during my trip to Los Angeles
 app.get('/api/images', function(req, res){
   res.json({
-    background: [
+    sameple_background: [
         {
         _id:  1,
         name: "Beach",
-        image_link: "/images/beach.jpg",
+        link: "/images/beach.jpg",
         },{
         _id:  2,
         name: "Snow Mountain",
-        image_link: "/images/snowmountain.jpg",
+        link: "/images/snowmountain.jpg",
 
         },{
         _id: 3,
         name: "Under Water",
-        image_link: "/images/underwater.jpg",
+        link: "/images/underwater.jpg",
         }, {
         _id: 4,
         name: "Mossholder",
-        image_link: "/images/mossholder.jpg",
+        link: "/images/mossholder.jpg",
+        }
+      ],
+    LA_trip: [
+        {
+        _id:  1,
+        name: "Adorable Baseball Dudes",
+        link: "/images/LA_trip/basball_dudes.jpg",
+        description: "These are some adorable toys I saw at one the LA stores"
+        },{
+        _id:  2,
+        name: "Delicious Dessert!",
+        link: "/images/LA_trip/casino_dessert.jpg",
+        description: "Some awesome dessert I had at Morongo Casino Resort"
+
+        },{
+        _id: 3,
+        name: "Chinese Theatre",
+        link: "/images/LA_trip/chinese_theatre.jpg",
+        description: "One of the popular lankmarks at the Hollywood Boulevard, Los Angeles"
+        }, {
+        _id: 4,
+        name: "Macaroon",
+        link: "/images/LA_trip/macaroon.jpg",
+        description: "What can I say? They are some soft little sweet treat!"
+        }, {
+        _id: 4,
+        name: "Oscar EVERYWHERE!",
+        link: "/images/LA_trip/oscar.jpg",
+        description: "Oscars I have achieved in my life haha. (Not)"
         }
       ]
   });
 });
 
-// Api template for videos
+// Api template for videos that i found online and its interesting
 app.get('/api/videos', function(req, res){
   res.json({
     samples: [
       {
         _id: 1,
         name: "Camping",
-        image_link: "/videos/camping.mp4",
+        link: "/videos/camping.mp4",
       },{
         _id: 2,
         name: "Dog in Sunset",
-        image_link: "/videos/dog_in_sunset.mp4",
+        link: "/videos/dog_in_sunset.mp4",
       },{
         _id: 3,
         name: "Sunset",
-        image_link: "/videos/underwater.mp4",
+        link: "/videos/underwater.mp4",
       },{
         _id: 4,
         name: "Underwater",
-        image_link: "/videos/underwater.mp4",
+        link: "/videos/underwater.mp4",
       }
```

Comment:
I see you have these videos here, but I am not sure where to find them on your API doc.